### PR TITLE
chore: edk based superset

### PR DESCRIPTION
**Pull request by pandemicsyn**
A couple note-able changes from the vanilla superset config

1. the `create-admin` commands isn't a direct call to superset anymore and gets proxied to the superset_ext
2. New explicit config option of `SUPERSET_CONFIG_PATH` that tells the extension where it should deploy the `superset_config.py` , and `SUPERSET_HOME` that tells gunicorn and superset where its rundir should be.

---
>  **Warning** 
> This PR depends on the below superset-ext pr being merged.

- https://github.com/meltano/superset-ext/pull/3

### CODE REVIEW

File: _data/meltano/utilities/superset_ext/apache.yml [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+  value: 60
+  description: Timeout in sections, used by `meltano invoke superset:ui`. Corresponds to the `gunicorn` `--timeout` option
+- name: ui.workers
+  label: UI Workers
+  value: 4
+  description: Workers used by `meltano invoke superset:ui`. Corresponds to the `gunicorn` `--workers` option
+- name: SUPERSET_CONFIG_PATH
+  label: Superset config path
+  description: The path to where the Superset config file is located.
+  value: $MELTANO_PROJECT_ROOT/superset/superset_config.py
```
Comment (pandemicsyn) [inline _data/meltano/utilities/superset_ext/apache.yml:31]: Airflow we ended up organizing under a `$MELTANO_PROJECT_ROOT/orchestrate/` , do we want to stash superset under something similar?

File: _data/meltano/utilities/superset_ext/apache.yml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+    This is equivalent to `superset fab create-admin` in the Superset documentation
+
+  1. Optionally, load some example data to play with:
+
+    ```sh
+    meltano invoke superset_ext:load-examples
+    ```
+
+    This is equivalent to `superset load_examples` in the Superset documentation
```
Comment (tayloramurphy) [inline _data/meltano/utilities/superset_ext/apache.yml:101]:
```suggestion
    This is equivalent to `superset load_examples` in the Superset documentation
    
    **Note that you need a user named `admin` with the admin permissions to load these examples.
```

File: _data/meltano/utilities/superset_ext/apache.yml [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+next_steps: |
+  1. If you're running Superset for the first time in a new environment you'll need to initialize the config, database, as well as generate generate a new [`SECRET_KEY`](#secret-key) to increase security:
+
+    ```sh
+    # create a new secret key
+    meltano config superset_ext set SECRET_KEY $(openssl rand -base64 42)
+    # create the initial config, database, and auth roles.
+    meltano invoke superset_ext:initialize
+    ```
+  1. Create an admin user: (use `admin` as username to be able to load the examples)
```
Comment (tayloramurphy) [inline _data/meltano/utilities/superset_ext/apache.yml:86]:
```suggestion
  2. Create an admin user: (use `admin` as username to be able to load the examples)
```

annoyingly it doesn't seem the markdown renders them as an ordered list

File: _data/meltano/utilities/superset_ext/apache.yml [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+
+    ```sh
+    # add a superset admin using prompting for required values
+    meltano invoke superset_ext:create-admin
+    ```
+
+    This is equivalent to `superset fab create-admin` in the Superset documentation
+
+  1. Optionally, load some example data to play with:
```
Comment (tayloramurphy) [inline _data/meltano/utilities/superset_ext/apache.yml:95]:
```suggestion
  3. Optionally, load some example data to play with:
```

File: _data/meltano/utilities/superset_ext/apache.yml [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+    ```sh
+    meltano invoke superset_ext:load-examples
+    ```
+
+    This is equivalent to `superset load_examples` in the Superset documentation
+    
+    **Note that you need a user named `admin` with the admin permissions to load these examples.**
+
+  1. Launch the Superset UI and log in using the username/password you created:
```
Comment (tayloramurphy) [inline _data/meltano/utilities/superset_ext/apache.yml:105]:
```suggestion
  4. Launch the Superset UI and log in using the username/password you created:
```

**Review by tayloramurphy — APPROVED** [review]

File: _data/default_variants.yml [hunk 6]
```diff
@@ -11,6 +11,7 @@ utilities:
   metabase: metabase
   sqlfluff: sqlfluff
   superset: apache
+  superset_ext: apache
```
Comment (aaronsteers) [inline _data/default_variants.yml:14]: I'm not sure if we want dupe `superset` and `superset_ext` entries in the long run here.

**Review by aaronsteers — CHANGES_REQUESTED** [review]
Suggesting we hold for discussion on this question.

File: _data/default_variants.yml (hunk 6 above)
Comment (tayloramurphy) [reply _data/default_variants.yml:14]: @aaronsteers I synced with Pat on this. I think we're going to make a `meltano` variant for the extension variant and then when the time comes we can deprecate it and change the default apache variant to be EDK based.

**Review by pnadolny13 — APPROVED** [review]
@tayloramurphy looks good to me!

**Pull request merged**